jenkins-build-driver/src/test/java/org/jboss/pnc/jenkinsbuilddriver/test/JenkinsDriverRemoteTest.java

```diff
@@ -143,12 +142,12 @@ public RepositorySession getRepositorySession() {
             
             @Override
             public String getJenkinsUrl() {
-                return System.getProperty("PNC_JENKINS_URL");
+                return System.getenv("PNC_JENKINS_URL") + ":" + getJenkinsPort();
```

Comment:
Matej, this is only a test usage, in real build you will get it from Docker Build driver. Test environment needs to be flexible and common configuration doesn't fit to this need IMO.